mlflow/store/tracking/sqlalchemy_store.py

```diff
@@ -515,6 +521,49 @@ def _get_run(self, session, run_uuid, eager=False):
 
         return runs[0]
 
+    def _get_run_inputs(self, session, run_uuids):
+        SqlDatasetAlias = sqlalchemy.orm.aliased(SqlDataset)
+        datasets = (
```

Comment:
Nit:

```suggestion
        dataset_inputs = (
```

Reply:
This should still be datasets. We are only fetching datasets in this call, and form the `dataset_inputs` once we have the input tags.

Reply:
You're right, whoops :)